StRoot/StETofCalibMaker/StETofCalibMaker.cxx

```diff
@@ -1100,15 +1100,32 @@ StETofCalibMaker::Make()
 {
     LOG_DEBUG << "StETofCalibMaker::Make(): starting ..." << endm;
 
-    mEvent = ( StEvent* ) GetInputDS( "StEvent" );
+    //mEvent = ( StEvent* ) GetInputDS( "StEvent" );
+    mEvent = NULL; //don't check for StEvent for genDst.C testing. PW
```

Comment:
I expect the commented line should be switched here for the commit.

Reply:
Corrected by the commit below

Reply:
Note, that this change is from an intermediate commit. Final version has the proper line update

Reply:
Sorry, my mistake. didn't see it correctly. Now solved with latest commit